envs/monkey_zoo/blackbox/utils/credential_config_generator/credential_config_generator.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import copy
+import json
+import os
+from pathlib import Path
+
+import nanoid as nanoid
+
+parser = argparse.ArgumentParser(
+    prog="Credential Config Generator",
+    description="Small script to add usernames and " "passwords to Infection Monkey configuration",
+)
+
+parser.add_argument(
+    "-u",
+    "--userlist-path",
+    type=str,
+    required=True,
+    help="Path to userlist file. Userfile should contain one username per line.",
+)
+parser.add_argument(
+    "-p",
+    "--passwordlist-path",
+    type=str,
+    required=True,
+    help="Path to passwordlist file. " "Passwordfile should contain one password per line.",
+)
+parser.add_argument(
+    "-c",
+    "--config-path",
+    type=str,
+    help="Path to Infection Monkey configuration file."
+    "Must be unencrypted. If not provided, will use default config from a file.",
```

Comment:
Where the default config is stored isn't relevant to the user.
```suggestion
    "Must be unencrypted. If not provided, default config will be used.",
```